package.json

```diff
@@ -1,24 +1,33 @@
 {
-	"name": "fbw-local-api",
+	"name": "fbw-simbridge",
 	"version": "0.2.4",
-	"description": "The local-api server for FBW addons for various tasks the addons itself can't achieve",
+	"description": "The simbridge server for FBW addons for various tasks the addons themselves can't achieve",
 	"author": "",
 	"private": false,
-	"license": "GNU",
+	"license": "GPL-3.0-only",
 	"os": [
 		"win32"
 	],
+	"engines": {
+		"node": "16.4.x",
+		"npm": "7.24.x"
```

Comment:
as mentioned previously, you need NPM version >=8.5.5, else it's just gonna fail the workflow.

![grafik](https://user-images.githubusercontent.com/50082450/170882808-c2dd43ad-f6c9-40b7-b23e-1a2f2ba82ad9.png)
(using exactly the same workflow as here until after set node version)